### PR TITLE
Update Simplified_Chinese.properties

### PR DESCRIPTION
**Pull request by xk730**

### CODE REVIEW

File: android/assets/jsons/translations/Simplified_Chinese.properties [hunk 1]
```diff
@@ -964,19 +960,13 @@ Worked by [cityName] = 被[cityName]开发
...
-Citizen Management = 
- # Requires translation!
-Avoid Growth = 
- # Requires translation!
-Default Focus = 
- # Requires translation!
-[stat] Focus = 
+Reset Citizens = 重新分配公民
+Citizen Management = 管理公民
+Avoid Growth = 停止增长
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Simplified_Chinese.properties:965]: Start a game and you will find that you cannot totally ‘stop' growth with this button. All you can do is to slow them down. So 人口控制或是减少增长会更好。

Comment (heipizhu4) [inline android/assets/jsons/translations/Simplified_Chinese.properties:965]: Start a game and you will find that you cannot totally ‘stop' growth with this button. All you can do is to slow them down. So 人口控制或是减少增长会更好。

File: android/assets/jsons/translations/Simplified_Chinese.properties [hunk 2]
```diff
@@ -964,19 +960,13 @@ Worked by [cityName] = 被[cityName]开发
...
+Default Focus = 默认的公民分配重点
+[stat] Focus = 目前分配重点：[stat]
```
Comment (heipizhu4) [inline android/assets/jsons/translations/Simplified_Chinese.properties:967]: image
从这张截图可以看到focus不是只有当前会这么写，而是一个选项。所以直接写'注重[stat]就可以了。

**Review by heipizhu4 — CHANGES_REQUESTED** [review]
2 errors. Maybe you should update your game before translating in order to avoid these mistakes?

File: android/assets/jsons/translations/Simplified_Chinese.properties (hunk 2 above)
Comment (xk730) [reply android/assets/jsons/translations/Simplified_Chinese.properties:967]: 谢谢更正；此外我一直有在更新游戏

**Review by heipizhu4 — APPROVED** [review]
Sorry for the wrong blame. Nice work now!

**Pull request merged**